src/components/AdminHome.js

```diff
@@ -1,22 +1,86 @@
 import React, { useState, useEffect } from 'react';
+import AddStudent from './AddStudent';
+import EditStudent from './EditStudent';
+import {SERVER_URL} from '../constants';
 
 const AdminHome = ()  => {
 
+  // function const fl = () => { fetchStudents(); }
+  // if you give it an empty list it will call the function one time
+
+  //useEffect(fetchStudents(), [] );
+
     useEffect(() => {
         // called once after intial render
         fetchStudents();
         }, [] )
 
+    /* 
+        *  list of students 
+    */
+    const [students, setStudents] = useState([]);
+    const [message, setMessage] = useState('');
 
     const fetchStudents = () => {
-		//TODO complete this method to fetch students and display list of students
+        console.log("fetchStudents ");
+        fetch(`${SERVER_URL}/student`)
+        .then((response) => response.json() )
+        .then((data) => setStudents(data) )
+        .catch((err) =>  { console.log("fetch error "+err); } );
+    }
+
+    const refreshStudents = () => {
+        setMessage('');
+        fetchStudents();
+    }
+
+    const deleteStudent = (event) => {
+        const row_id = event.target.parentNode.parentNode.rowIndex - 1;
```

Comment:
This parentNode logic works, but creates a dependency in the function and should probably be avoided.